### PR TITLE
Add support for on/off switch labels when built on iOS 13.

### PR DESCRIPTION
**Pull request by brandondiamond**
Same as #12404 but verifies platform is iOS 13. Shouldn't merge until build bots have been updated to support iOS 13.

### CODE REVIEW

File: shell/platform/fuchsia/runtime/dart/utils/BUILD.gn [hunk 1]
```diff
@@ -34,10 +34,10 @@ source_set("utils") {
     "$fuchsia_sdk_root/pkg:async-loop-default",
     "$fuchsia_sdk_root/pkg:fdio",
     "$fuchsia_sdk_root/pkg:memfs",
-    "$fuchsia_sdk_root/pkg:syslog",
-    "$fuchsia_sdk_root/pkg:zx",
     "$fuchsia_sdk_root/pkg:sys_cpp",
+    "$fuchsia_sdk_root/pkg:syslog",
     "$fuchsia_sdk_root/pkg:vfs_cpp",
+    "$fuchsia_sdk_root/pkg:zx",
```
Comment (brandondiamond) [inline shell/platform/fuchsia/runtime/dart/utils/BUILD.gn]: This was altered by the formatting script. Not sure if I should revert or leave alone.

Comment (chinmaygarde) [reply shell/platform/fuchsia/runtime/dart/utils/BUILD.gn]: No. Leave it be. This is good. We format all our GN files. I believe this one was imported from //topaz as-is recently.

**Comment by chinmaygarde** [comment]
Migration to the latest SDK is pending in https://github.com/flutter/infra/pull/62

**Comment by johnsonmh** [comment]
> Migration to the latest SDK is pending in [flutter/infra#62](https://github.com/flutter/infra/pull/62)

@chinmaygarde is this PR blocked on https://github.com/flutter/infra/pull/62 ? I know https://github.com/flutter/infra/pull/62 is blocked on the device lab updates, but it's not clear to me that this PR needs to wait for those changes, as it wouldn't break anything pre iOS13 right?

**Comment by cbracken** [comment]
flutter/infra#62 has landed, but the devices are still iOS 12. It's not necessary to wait for them to be upgraded, but we do need the engine builder recipes migrated to Xcode 11 with the iOS 13 SDK. 

I believe @chinmaygarde made one attempt at that but that ended up being reverted. Once that's re-landed, this should be landable.

**Comment by LongCatIsLooong** [comment]
@brandondiamond https://github.com/flutter/infra/pull/62 is relanded.

**Comment by brandondiamond** [comment]
🎉 that's great news! thanks for everything.

**Comment by cbracken** [comment]
Merged. Fingers crossed!

**Pull request merged**